x-pack/platform/packages/shared/kbn-langgraph-checkpoint-saver/server/elastic-search-checkpoint-saver/index.ts

```diff
@@ -375,20 +413,45 @@ export class ElasticSearchSaver extends BaseCheckpointSaver {
         type,
       };
 
+      this.logger.debug(
+        `Indexing write operation for checkpoint ${checkpointId}: ${JSON.stringify(doc)}`
```

Comment:
is it possible that `doc` has some sensitive information which we should not print?